XenAdmin/Controls/XenSearch/SearchOutput.resx

```diff
@@ -218,6 +218,7 @@
   </data>
   <data name="buttonColumns.Text" xml:space="preserve">
     <value>Sh&amp;ow Columns</value>

```

Comment:
Oops you're right, I think that's from cherry picking the line in the diff - I'll remove the old value